### PR TITLE
🌐 [i18n-KO] Translated `image_to_image.md` to Korean

### PR DESCRIPTION
**Pull request by shinhyunji36**
# What does this PR do?

Translated the `image_to_image.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@jun048098, @yijun-lee, @mreraser, @shinhyunji36, @heuristicwave

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/tasks/image_to_image.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+[[open-in-colab]]
+
+Image-to-Image 작업은 애플리케이션이 이미지를 입력받아 또 다른 이미지를 출력하는 작업입니다. 여기에는 이미지 향상(초고해상도, 저조도 향상, 빗줄기 제거 등), 이미지 복원 등 다양한 하위 작업이 포함됩니다.
+
+이 가이드에서는 다음을 수행하는 방법을 보여줍니다.
+- 초고해상도 작업을 위한 image-to-image 파이프라인 사용,
+- 파이프라인 없이 동일한 작업을 위한 image-to-image 모델 실행
+
+이 가이드가 발표된 시점에서는, `image-to-image` 파이프라인은 초고해상도 작업만 지원합니다.
```
Comment (heuristicwave) [inline docs/source/ko/tasks/image_to_image.md:27]:
```suggestion
이 가이드가 발표된 시점에서는, `image-to-image` 파이프라인은 초고해상도 작업만 지원한다는 점을 유의하세요.
```
Note that의 의미를 살려 조금 변경해보았습니다.

File: docs/source/ko/tasks/image_to_image.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+파이프라인 없이 직접 추론을 수행하려면 Transformers의 `Swin2SRForImageSuperResolution` 및 `Swin2SRImageProcessor` 클래스를 사용할 수 있습니다. 이를 위해 동일한 모델 체크포인트를 사용합니다. 모델과 프로세서를 초기화해 보겠습니다. 
+
+```python
+from transformers import Swin2SRForImageSuperResolution, Swin2SRImageProcessor 
+
+model = Swin2SRForImageSuperResolution.from_pretrained("caidas/swin2SR-lightweight-x2-64").to(device)
+processor = Swin2SRImageProcessor("caidas/swin2SR-lightweight-x2-64")
+```
+
+파이프라인은 우리가 직접 해야 하는 전처리와 후처리 단계를 추상화해 주므로, 이제 이미지를 전처리해 봅시다. 이미지를 프로세서에 전달한 다음, 픽셀 값을 GPU로 이동시킬 것입니다.
```
Comment (heuristicwave) [inline docs/source/ko/tasks/image_to_image.md:81]:
```suggestion
`pipeline` 우리가 직접 수행해야 하는 전처리와 후처리 단계를 추상화하므로, 이미지를 전처리해 보겠습니다. 이미지를 프로세서에 전달한 다음 픽셀 값을 GPU로 이동시키겠습니다. 
```
원문에서 `pipeline` 라고 표현된 부분과, 문장을 조금 더 매끄럽게 바꿔보았습니다.

File: docs/source/ko/tasks/image_to_image.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+outputs.reconstruction.data.shape
+# torch.Size([1, 3, 880, 1072])
+```
+
+출력 텐서의 차원을 축소하고 0번째 축을 제거한 다음, 값을 클리핑하고 NumPy 부동소수점 배열로 변환해야합니다. 그런 다음 [1072, 880] 모양을 갖도록 축을 재정렬하고 마지막으로 출력을 0과 255 사이의 값을 갖도록 되돌립니다.
+
+```python
+import numpy as np
+
+# 크기를 줄이고, CPU로 이동하고, 값을 클리핑합니다.
```
Comment (heuristicwave) [inline docs/source/ko/tasks/image_to_image.md:122]:
```suggestion
# 크기를 줄이고, CPU로 이동하고, 값을 클리핑
```
영어 원문과 비슷하게 코멘트 형식의 느낌으로 변경했습니다.

File: docs/source/ko/tasks/image_to_image.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+output = outputs.reconstruction.data.squeeze().cpu().clamp_(0, 1).numpy()
+# 축을 재정렬합니다.
```
Comment (heuristicwave) [inline docs/source/ko/tasks/image_to_image.md:124]:
```suggestion
# 축을 재정렬
```

File: docs/source/ko/tasks/image_to_image.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+output = np.moveaxis(output, source=0, destination=-1)
+# 값을 픽셀값 범위로 되돌립니다.
```
Comment (heuristicwave) [inline docs/source/ko/tasks/image_to_image.md:126]:
```suggestion
# 값을 픽셀값 범위로 되돌리기
```

**Review by heuristicwave — COMMENTED** [review]
@shinhyunji36 리뷰 완료했습니다! Before reviewing 체크리스트도 체크해주세요!

File: docs/source/ko/tasks/image_to_image.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+       device='cuda:0'), hidden_states=None, attentions=None)
+```
+`reconstruction`를 가져와 시각화를 위해 후처리해야 합니다. 어떻게 생겼는지 살펴봅시다.
+
+```python
+outputs.reconstruction.data.shape
+# torch.Size([1, 3, 880, 1072])
+```
+
+출력 텐서의 차원을 축소하고 0번째 축을 제거한 다음, 값을 클리핑하고 NumPy 부동소수점 배열로 변환해야합니다. 그런 다음 [1072, 880] 모양을 갖도록 축을 재정렬하고 마지막으로 출력을 0과 255 사이의 값을 갖도록 되돌립니다.
```
Comment (mreraser) [inline docs/source/ko/tasks/image_to_image.md:117]:
```suggestion
출력 텐서의 차원을 축소하고 0번째 축을 제거한 다음, 값을 클리핑하고 NumPy 부동소수점 배열로 변환해야 합니다. 그런 다음 [1072, 880] 모양을 갖도록 축을 재정렬하고 마지막으로 출력을 0과 255 사이의 값을 갖도록 되돌립니다.
```

맞춤법으로 한 칸 띄우는걸 제안드립니다.

File: docs/source/ko/tasks/image_to_image.md (hunk 2 above)
Comment (mreraser) [inline docs/source/ko/tasks/image_to_image.md:81]:
```suggestion
파이프라인은 우리가 직접 해야 하는 전처리와 후처리 단계를 추상화해 주므로, 이제 이미지를 전처리해 봅시다. 이미지를 프로세서에 전달한 다음, 픽셀값을 GPU로 이동시킬 것입니다. 
```

[해당 문헌](https://wikidocs.net/80784)을 참고하여 픽셀과 값 사이의 띄어쓰기를 제거하는 것을 제안드립니다.

File: docs/source/ko/tasks/image_to_image.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+파이프라인은 우리가 직접 해야 하는 전처리와 후처리 단계를 추상화해 주므로, 이제 이미지를 전처리해 봅시다. 이미지를 프로세서에 전달한 다음, 픽셀 값을 GPU로 이동시킬 것입니다. 
+
+```python
+pixel_values = processor(image, return_tensors="pt").pixel_values
+print(pixel_values.shape)
+
+pixel_values = pixel_values.to(device)
+```
+
+이제 픽셀 값을 모델에 전달하여 이미지를 추론할 수 있습니다.
```
Comment (mreraser) [inline docs/source/ko/tasks/image_to_image.md:90]:
```suggestion
이제 픽셀값을 모델에 전달하여 이미지를 추론할 수 있습니다.
```

**Comment by mreraser** [comment]
번역하느라 고생하셨습니다!😄 @shinhyunji36

**Review by yijun-lee — APPROVED** [review]
LGTM

File: docs/source/ko/tasks/image_to_image.md (hunk 7 above)
Comment (shinhyunji36) [reply docs/source/ko/tasks/image_to_image.md:90]: https://github.com/huggingface/transformers/pull/32327/files#r1703201649 이 부분과 함께 처리하기 위해 잠시 보류하겠습니다.

File: docs/source/ko/tasks/image_to_image.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+파이프라인 없이 직접 추론을 수행하려면 Transformers의 `Swin2SRForImageSuperResolution` 및 `Swin2SRImageProcessor` 클래스를 사용할 수 있습니다. 이를 위해 동일한 모델 체크포인트를 사용합니다. 모델과 프로세서를 초기화해 보겠습니다. 
+
+```python
+from transformers import Swin2SRForImageSuperResolution, Swin2SRImageProcessor 
+
+model = Swin2SRForImageSuperResolution.from_pretrained("caidas/swin2SR-lightweight-x2-64").to(device)
+processor = Swin2SRImageProcessor("caidas/swin2SR-lightweight-x2-64")
+```
+
+`pipeline` 우리가 직접 수행해야 하는 전처리와 후처리 단계를 추상화하므로, 이미지를 전처리해 보겠습니다. 이미지를 프로세서에 전달한 다음 픽셀 값을 GPU로 이동시키겠습니다.
```
Comment (shinhyunji36) [inline docs/source/ko/tasks/image_to_image.md:81]:
```suggestion
`pipeline` 우리가 직접 수행해야 하는 전처리와 후처리 단계를 추상화하므로, 이미지를 전처리해 보겠습니다. 이미지를 프로세서에 전달한 다음 픽셀값을 GPU로 이동시키겠습니다. 
```

**Review by stevhliu — APPROVED** [review]
LGTM, thanks for your translation! 🤗

**Pull request merged**